nncf/torch/model_creation.py

```diff
@@ -309,3 +309,23 @@ def create_compression_algorithm_builder_from_algo_names(
     else:
         builder = PTCompositeCompressionAlgorithmBuilder(config, should_init=should_init)
     return builder
+
+
+def wrap_model(model: torch.nn.Module, example_input: Any) -> NNCFNetwork:
+    """
+    Creates NNCFNetwork instance for the PyTorch model where the first item of dataset
```

Comment:
Looks like to be correct, description should be like
```
    """Wraps a PyTorch model to the NNCFNetwork class.

    This function dynamically extends the instance of PyTorch model with NNCF-enabling functionality. 

    :param model: PyTorch model.
...
```
And missed dot for param.

Reply:
done